src/main/kotlin/org/rust/ide/formatter/impl/spacing.kt

```diff
@@ -268,6 +272,11 @@ private fun SpacingContext.blockMustBeMultiLine(): Boolean {
     }
 }
 
+private fun SpacingContext.needsBlankLineBetweenItems(): Boolean {
+    // Allow to keep consecutive runs of `use`, `const` or `static` items without blank lines
+    return !(elementType1 == elementType2 && elementType1 in ONE_LINE_ITEMS)
```

Comment:
Here is how Kotlin attaches comments:

```
class Foo

//bar
/*bar */ class Bar //bar
//self standing comment

//baz
class Baz
```

Looks like in the end we would wont *both* comment binders and special comment handling =/